powdr_cli/src/main.rs

```diff
@@ -1,10 +1,21 @@
 //! The powdr CLI tool
 
+mod util;
+
 use clap::{Parser, Subcommand};
 use env_logger::{Builder, Target};
 use log::LevelFilter;
-use number::{FieldElement, GoldilocksField};
+use number::{Bn254Field, FieldElement, GoldilocksField};
 use std::{fs, io::Write, path::Path};
+use strum::{Display, EnumString, EnumVariantNames};
+
+#[derive(Clone, EnumString, EnumVariantNames, Display)]
+pub enum FieldArgument {
+    #[strum(serialize = "gl")]
+    Gl,
+    #[strum(serialize = "bn128")]
```

Comment:
halo2 calls it BN256, we call it BN254, and here it's bn128 haha
why 128 here?

Reply:
Right haha 128 is the security, 254 is the field size 256 is ???. I forgot I called it 254 in the fields, so maybe we should go with 254

Reply:
Done